evm/utils.js

```diff
@@ -60,6 +60,10 @@ const IDeployer = require('@axelar-network/axelar-gmp-sdk-solidity/interfaces/ID
 const { exec } = require('child_process');
 const { verifyContract } = require(`${__dirname}/../axelar-chains-config`);
 
+const getITSHubAddress = (config) => {
```

Comment:
avoid get prefix for getters. Also, add this to `common/utils.js` since it's not EVM specific, and we already have other generic ITS helpers there